src/main/java/org/openhab/tools/analysis/checkstyle/AboutHtmlCheck.java

```diff
@@ -26,30 +28,35 @@
 
 /**
  * Checks if an about.html file is valid.
+ * Verifies if the about.html file is added to the build.properties file
  *
- * @author Petar Valchev
- *
+ * @author Petar Valchev - Initial Implementation
+ * @author Svilen Valkanov - Add check for inclusion in build.properties file
  */
 public class AboutHtmlCheck extends AbstractStaticCheck {
```

Comment:
We have chosen this approach, keep everything that is related with the about.html at single place. In other checks (e.g. service component check) this makes more sense and we we prefer to keep it consistent.